### ISSUE TITLE
area that drops the animation

### ISSUE THREAD

**Issue by JenaPlinsky** [psifidotos/nowdock-panel#60]
there is an area between the base of the icons and the edge of the screen, that if through with the mouse, it drops the animation making it less smooth
is it correct?

**Comment by psifidotos** [comment]
Yes, this must be true, this is a hidden plasma window... It we change
architecture for version 0.6 this should be fixed...

Στις 21 Δεκ 2016 20:41, ο χρήστης "JenaPlinsky" <notifications@github.com>
έγραψε:

**Comment by JenaPlinsky** [comment]
Awesome !!!
